### PR TITLE
chore(ci): enforce `msrv` in parts of our CI.

### LINKED ISSUE #4874: Configure and check Minimum Supported Rust Version
https://github.com/oxc-project/oxc/blob/a2269625cc3cef3252c821018536b048deaba037/Cargo.toml#L15

It's 1.74 but seems not enforced?

### PR DESCRIPTION
**Pull request by rzvxa**
closes #4874 

image

### CODE REVIEW

**Comment by rzvxa** [comment]
* **#4879** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4879?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rzvxa and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by Boshen** [comment]
Clever!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -67,7 +67,8 @@ jobs:
           RUSTUP_HOME: ${{ env.DEV_DRIVE }}/.rustup
         shell: bash
         run: |
-          sed -i -e 's/profile = "default"/profile = "minimal"/g' rust-toolchain.toml
+          MSRV=$(awk -F'=' '/rust-version/ {gsub(/[" ]/, "", $2); printf "%s", ($2 + "")}' Cargo.toml) # NOTE: this will fail if there are any other items named `rust-version`.
```
[the remark this answers is not in the document]
Comment (rzvxa) [reply .github/workflows/ci.yml]: Why do I get this?

image

[the remark this answers is not in the document]
Comment (rzvxa) [reply .github/workflows/ci.yml]: I guess it has nothing to do with us😕

**Pull request merged**